agcaballero/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -33,54 +33,122 @@
    */ 
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
     Collection<String> attendees = request.getAttendees();
-    ArrayList<TimeRange> times = new ArrayList<TimeRange>();
-     // minutes of day is + 1 to account for first and last minute of day
-    boolean[] minutes = new boolean[MINUTES_IN_DAY + 1];
-
-    for(int i = 0; i < minutes.length; i++) {
-      minutes[i] = true;
+    Collection<String> optionalAttendees = request.getOptionalAttendees();
+    
+    // are there any mandatory employees?
+    boolean mandatory = false;
```

Comment:
Consider naming `hasMandatoryAttendees` for clarity.

Reply:
I will change that. Thank you!

Reply:
Done